opentelemetry/daemonset/otelcol.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+metadata:
+  name: daemonset
```

Comment:
We should think of a better name this will created `daemonset-collector` which is quite vague.